// @clerk/localizations/dist/ja-JP.js
"use strict";
var __defProp = Object.defineProperty;
var __getOwnPropDesc = Object.getOwnPropertyDescriptor;
var __getOwnPropNames = Object.getOwnPropertyNames;
var __hasOwnProp = Object.prototype.hasOwnProperty;
var __export = (target, all) => {
  for (var name in all)
    __defProp(target, name, { get: all[name], enumerable: true });
};
var __copyProps = (to, from, except, desc) => {
  if (from && typeof from === "object" || typeof from === "function") {
    for (let key of __getOwnPropNames(from))
      if (!__hasOwnProp.call(to, key) && key !== except)
        __defProp(to, key, { get: () => from[key], enumerable: !(desc = __getOwnPropDesc(from, key)) || desc.enumerable });
  }
  return to;
};
var __toCommonJS = (mod) => __copyProps(__defProp({}, "__esModule", { value: true }), mod);

// src/ja-JP.ts
var ja_JP_exports = {};
__export(ja_JP_exports, {
  jaJP: () => jaJP
});
module.exports = __toCommonJS(ja_JP_exports);
var jaJP = {
  locale: "ja-JP",
  __experimental_userVerification: {
    alternativeMethods: {
      actionLink: void 0,
      actionText: void 0,
      blockButton__backupCode: void 0,
      blockButton__emailCode: void 0,
      blockButton__password: void 0,
      blockButton__phoneCode: void 0,
      blockButton__totp: void 0,
      getHelp: {
        blockButton__emailSupport: void 0,
        content: void 0,
        title: void 0
      },
      subtitle: void 0,
      title: void 0
    },
    backupCodeMfa: {
      subtitle: void 0,
      title: void 0
    },
    emailCode: {
      formTitle: void 0,
      resendButton: void 0,
      subtitle: void 0,
      title: void 0
    },
    noAvailableMethods: {
      message: void 0,
      subtitle: void 0,
      title: void 0
    },
    password: {
      actionLink: void 0,
      subtitle: void 0,
      title: void 0
    },
    phoneCode: {
      formTitle: void 0,
      resendButton: void 0,
      subtitle: void 0,
      title: void 0
    },
    phoneCodeMfa: {
      formTitle: void 0,
      resendButton: void 0,
      subtitle: void 0,
      title: void 0
    },
    totpMfa: {
      formTitle: void 0,
      subtitle: void 0,
      title: void 0
    }
  },
  backButton: "\u623B\u308B",
  badge__default: "\u30C7\u30D5\u30A9\u30EB\u30C8",
  badge__otherImpersonatorDevice: "\u4ED6\u306E\u6A21\u5023\u8005\u30C7\u30D0\u30A4\u30B9",
  badge__primary: "\u30D7\u30E9\u30A4\u30DE\u30EA",
  badge__requiresAction: "\u30A2\u30AF\u30B7\u30E7\u30F3\u304C\u5FC5\u8981",
  badge__thisDevice: "\u3053\u306E\u30C7\u30D0\u30A4\u30B9",
  badge__unverified: "\u672A\u78BA\u8A8D",
  badge__userDevice: "\u30E6\u30FC\u30B6\u30FC\u30C7\u30D0\u30A4\u30B9",
  badge__you: "\u3042\u306A\u305F",
  createOrganization: {
    formButtonSubmit: "\u7D44\u7E54\u3092\u4F5C\u6210\u3059\u308B",
    invitePage: {
      formButtonReset: "\u30B9\u30AD\u30C3\u30D7"
    },
    title: "\u7D44\u7E54\u306E\u4F5C\u6210"
  },
  dates: {
    lastDay: "\u6628\u65E5\u306E{{ date | timeString('ja-JP') }}\u306B",
    next6Days: "{{ date | weekday('ja-JP','long') }}\u306E{{ date | timeString('ja-JP') }}\u306B",
    nextDay: "\u660E\u65E5\u306E{{ date | timeString('ja-JP') }}\u306B",
    numeric: "{{ date | numeric('ja-JP') }}\u306B",
    previous6Days: "{{ date | weekday('ja-JP','long') }}\u306E{{ date | timeString('ja-JP') }}\u306B",
    sameDay: "\u4ECA\u65E5\u306E{{ date | timeString('ja-JP') }}\u306B"
  },
  dividerText: "\u307E\u305F\u306F",
  footerActionLink__useAnotherMethod: "\u5225\u306E\u65B9\u6CD5\u3092\u4F7F\u7528\u3059\u308B",
  footerPageLink__help: "\u30D8\u30EB\u30D7",
  footerPageLink__privacy: "\u30D7\u30E9\u30A4\u30D0\u30B7\u30FC",
  footerPageLink__terms: "\u5229\u7528\u898F\u7D04",
  formButtonPrimary: "\u7D9A\u3051\u308B",
  formButtonPrimary__verify: "\u78BA\u8A8D\u3059\u308B",
  formFieldAction__forgotPassword: "\u30D1\u30B9\u30EF\u30FC\u30C9\u3092\u304A\u5FD8\u308C\u3067\u3059\u304B\uFF1F",
  formFieldError__matchingPasswords: "\u30D1\u30B9\u30EF\u30FC\u30C9\u304C\u4E00\u81F4\u3057\u307E\u3059\u3002",
  formFieldError__notMatchingPasswords: "\u30D1\u30B9\u30EF\u30FC\u30C9\u304C\u4E00\u81F4\u3057\u307E\u305B\u3093\u3002",
  formFieldError__verificationLinkExpired: "\u691C\u8A3C\u30EA\u30F3\u30AF\u306E\u6709\u52B9\u671F\u9650\u304C\u5207\u308C\u3066\u3044\u307E\u3059\u3002\u65B0\u3057\u3044\u30EA\u30F3\u30AF\u3092\u30EA\u30AF\u30A8\u30B9\u30C8\u3057\u3066\u304F\u3060\u3055\u3044\u3002",
  formFieldHintText__optional: "\u4EFB\u610F",
  formFieldHintText__slug: "\u30B9\u30E9\u30B0\u306F\u4EBA\u9593\u304C\u8AAD\u3081\u308B\u30E6\u30CB\u30FC\u30AF\u306AID\u3067\u3059\u3002URL\u3067\u826F\u304F\u4F7F\u308F\u308C\u307E\u3059\u3002",
  formFieldInputPlaceholder__backupCode: "\u30D0\u30C3\u30AF\u30A2\u30C3\u30D7\u30B3\u30FC\u30C9",
  formFieldInputPlaceholder__confirmDeletionUserAccount: "\u30A2\u30AB\u30A6\u30F3\u30C8\u524A\u9664",
  formFieldInputPlaceholder__emailAddress: "\u30E1\u30FC\u30EB\u30A2\u30C9\u30EC\u30B9",
  formFieldInputPlaceholder__emailAddress_username: "\u30E1\u30FC\u30EB\u30A2\u30C9\u30EC\u30B9\u307E\u305F\u306F\u30E6\u30FC\u30B6\u30FC\u540D",
  formFieldInputPlaceholder__emailAddresses: "\u30B9\u30DA\u30FC\u30B9\u307E\u305F\u306F\u30AB\u30F3\u30DE\u3067\u533A\u5207\u3063\u3066\u30011\u3064\u4EE5\u4E0A\u306E\u30E1\u30FC\u30EB\u30A2\u30C9\u30EC\u30B9\u3092\u5165\u529B\u307E\u305F\u306F\u8CBC\u308A\u4ED8\u3051\u3066\u304F\u3060\u3055\u3044",
  formFieldInputPlaceholder__firstName: "\u540D",
  formFieldInputPlaceholder__lastName: "\u59D3",
  formFieldInputPlaceholder__organizationDomain: "\u7D44\u7E54\u30C9\u30E1\u30A4\u30F3",
  formFieldInputPlaceholder__organizationDomainEmailAddress: "\u7D44\u7E54\u30C9\u30E1\u30A4\u30F3\u306E\u30E1\u30FC\u30EB\u30A2\u30C9\u30EC\u30B9",
  formFieldInputPlaceholder__organizationName: "\u7D44\u7E54\u540D",
  formFieldInputPlaceholder__organizationSlug: "my-organization",
  formFieldInputPlaceholder__password: "\u30D1\u30B9\u30EF\u30FC\u30C9",
  formFieldInputPlaceholder__phoneNumber: "\u96FB\u8A71\u756A\u53F7",
  formFieldInputPlaceholder__username: "\u30E6\u30FC\u30B6\u30FC\u540D",
  formFieldLabel__automaticInvitations: "\u3053\u306E\u30C9\u30E1\u30A4\u30F3\u306E\u81EA\u52D5\u62DB\u5F85\u3092\u6709\u52B9\u306B\u3059\u308B",
  formFieldLabel__backupCode: "\u30D0\u30C3\u30AF\u30A2\u30C3\u30D7\u30B3\u30FC\u30C9",
  formFieldLabel__confirmDeletion: "\u524A\u9664\u306E\u78BA\u8A8D",
  formFieldLabel__confirmPassword: "\u30D1\u30B9\u30EF\u30FC\u30C9\u306E\u78BA\u8A8D",
  formFieldLabel__currentPassword: "\u73FE\u5728\u306E\u30D1\u30B9\u30EF\u30FC\u30C9",
  formFieldLabel__emailAddress: "\u30E1\u30FC\u30EB\u30A2\u30C9\u30EC\u30B9",
  formFieldLabel__emailAddress_username: "\u30E1\u30FC\u30EB\u30A2\u30C9\u30EC\u30B9\u307E\u305F\u306F\u30E6\u30FC\u30B6\u30FC\u540D",
  formFieldLabel__emailAddresses: "\u30E1\u30FC\u30EB\u30A2\u30C9\u30EC\u30B9",
  formFieldLabel__firstName: "\u540D",
  formFieldLabel__lastName: "\u59D3",
  formFieldLabel__newPassword: "\u65B0\u3057\u3044\u30D1\u30B9\u30EF\u30FC\u30C9",
  formFieldLabel__organizationDomain: "\u30C9\u30E1\u30A4\u30F3",
  formFieldLabel__organizationDomainDeletePending: "\u4FDD\u7559\u4E2D\u306E\u62DB\u5F85\u3068\u63D0\u6848\u3092\u524A\u9664",
  formFieldLabel__organizationDomainEmailAddress: "\u78BA\u8A8D\u7528\u306E\u30E1\u30FC\u30EB\u30A2\u30C9\u30EC\u30B9",
  formFieldLabel__organizationDomainEmailAddressDescription: "\u3053\u306E\u30C9\u30E1\u30A4\u30F3\u3092\u78BA\u8A8D\u3059\u308B\u305F\u3081\u306E\u30B3\u30FC\u30C9\u3092\u53D7\u3051\u53D6\u308B\u30E1\u30FC\u30EB\u30A2\u30C9\u30EC\u30B9\u3092\u5165\u529B\u3057\u3066\u304F\u3060\u3055\u3044\u3002",
  formFieldLabel__organizationName: "\u7D44\u7E54\u540D",
  formFieldLabel__organizationSlug: "\u30B9\u30E9\u30B0URL",
  formFieldLabel__passkeyName: void 0,
  formFieldLabel__password: "\u30D1\u30B9\u30EF\u30FC\u30C9",
  formFieldLabel__phoneNumber: "\u96FB\u8A71\u756A\u53F7",
  formFieldLabel__role: "\u5F79\u5272",
  formFieldLabel__signOutOfOtherSessions: "\u4ED6\u306E\u30C7\u30D0\u30A4\u30B9\u304B\u3089\u30B5\u30A4\u30F3\u30A2\u30A6\u30C8",
  formFieldLabel__username: "\u30E6\u30FC\u30B6\u30FC\u540D",
  impersonationFab: {
    action__signOut: "\u30B5\u30A4\u30F3\u30A2\u30A6\u30C8",
    title: "{{identifier}}\u3068\u3057\u3066\u30B5\u30A4\u30F3\u30A4\u30F3\u4E2D"
  },
  maintenanceMode: void 0,
  membershipRole__admin: "\u7BA1\u7406\u8005",
  membershipRole__basicMember: "\u30E1\u30F3\u30D0\u30FC",
  membershipRole__guestMember: "\u30B2\u30B9\u30C8",
  organizationList: {
    action__createOrganization: "\u7D44\u7E54\u3092\u4F5C\u6210\u3059\u308B",
    action__invitationAccept: "\u53C2\u52A0\u3059\u308B",
    action__suggestionsAccept: "\u53C2\u52A0\u3092\u30EA\u30AF\u30A8\u30B9\u30C8\u3059\u308B",
    createOrganization: "\u7D44\u7E54\u3092\u4F5C\u6210",
    invitationAcceptedLabel: "\u53C2\u52A0\u3057\u307E\u3057\u305F",
    subtitle: "{{applicationName}}",
    suggestionsAcceptedLabel: "\u627F\u8A8D\u5F85\u3061",
    title: "\u30A2\u30AB\u30A6\u30F3\u30C8\u3092\u9078\u629E",
    titleWithoutPersonal: "\u7D44\u7E54\u3092\u9078\u629E"
  },
  organizationProfile: {
    badge__automaticInvitation: "\u81EA\u52D5\u62DB\u5F85",
    badge__automaticSuggestion: "\u81EA\u52D5\u30B5\u30B8\u30A7\u30B9\u30C8",
    badge__manualInvitation: "\u81EA\u52D5\u767B\u9332\u306A\u3057",
    badge__unverified: "\u672A\u627F\u8A8D",
    createDomainPage: {
      subtitle: "\u30C9\u30E1\u30A4\u30F3\u3092\u8FFD\u52A0\u3057\u3066\u691C\u8A3C\u3057\u307E\u3059\u3002\u3053\u306E\u30C9\u30E1\u30A4\u30F3\u306E\u30E1\u30FC\u30EB\u30A2\u30C9\u30EC\u30B9\u3092\u6301\u3064\u30E6\u30FC\u30B6\u30FC\u306F\u3001\u81EA\u52D5\u7684\u306B\u7D44\u7E54\u306B\u53C2\u52A0\u3059\u308B\u304B\u3001\u53C2\u52A0\u3092\u30EA\u30AF\u30A8\u30B9\u30C8\u3059\u308B\u3053\u3068\u304C\u3067\u304D\u307E\u3059\u3002",
      title: "\u30C9\u30E1\u30A4\u30F3\u3092\u8FFD\u52A0"
    },
    invitePage: {
      detailsTitle__inviteFailed: "\u62DB\u5F85\u72B6\u3092\u9001\u4FE1\u3067\u304D\u307E\u305B\u3093\u3067\u3057\u305F\u3002\u4EE5\u4E0B\u3092\u4FEE\u6B63\u3057\u3066\u3082\u3046\u4E00\u5EA6\u8A66\u3057\u3066\u304F\u3060\u3055\u3044:",
      formButtonPrimary__continue: "\u62DB\u5F85\u72B6\u3092\u9001\u4FE1\u3059\u308B",
      selectDropdown__role: "\u5F79\u5272\u3092\u9078\u629E",
      subtitle: "\u3053\u306E\u7D44\u7E54\u306B\u65B0\u3057\u3044\u30E1\u30F3\u30D0\u30FC\u3092\u62DB\u5F85\u3059\u308B",
      successMessage: "\u62DB\u5F85\u72B6\u304C\u6B63\u5E38\u306B\u9001\u4FE1\u3055\u308C\u307E\u3057\u305F",
      title: "\u30E1\u30F3\u30D0\u30FC\u3092\u62DB\u5F85"
    },
    membersPage: {
      action__invite: "\u62DB\u5F85",
      activeMembersTab: {
        menuAction__remove: "\u30E1\u30F3\u30D0\u30FC\u306E\u524A\u9664",
        tableHeader__actions: void 0,
        tableHeader__joined: "\u53C2\u52A0\u65E5\u6642",
        tableHeader__role: "\u5F79\u5272",
        tableHeader__user: "\u30E6\u30FC\u30B6\u30FC"
      },
      detailsTitle__emptyRow: "\u8868\u793A\u3059\u308B\u30E1\u30F3\u30D0\u30FC\u306F\u3042\u308A\u307E\u305B\u3093",
      invitationsTab: {
        autoInvitations: {
          headerSubtitle: "\u30E1\u30FC\u30EB\u30C9\u30E1\u30A4\u30F3\u3092\u7D44\u7E54\u306B\u63A5\u7D9A\u3059\u308B\u3053\u3068\u3067\u30E6\u30FC\u30B6\u30FC\u3092\u62DB\u5F85\u3057\u307E\u3059\u3002\u4E00\u81F4\u3059\u308B\u30E1\u30FC\u30EB\u30C9\u30E1\u30A4\u30F3\u3092\u6301\u3064\u30E6\u30FC\u30B6\u30FC\u306F\u3001\u3044\u3064\u3067\u3082\u7D44\u7E54\u306B\u53C2\u52A0\u3059\u308B\u3053\u3068\u304C\u3067\u304D\u307E\u3059\u3002",
          headerTitle: "\u81EA\u52D5\u62DB\u5F85",
          primaryButton: "\u691C\u8A3C\u6E08\u307F\u30C9\u30E1\u30A4\u30F3\u3092\u7BA1\u7406"
        },
        table__emptyRow: "\u8868\u793A\u3059\u308B\u62DB\u5F85\u306F\u3042\u308A\u307E\u305B\u3093"
      },
      invitedMembersTab: {
        menuAction__revoke: "\u62DB\u5F85\u3092\u53D6\u308A\u6D88\u3059",
        tableHeader__invited: "\u62DB\u5F85\u6E08\u307F"
      },
      requestsTab: {
        autoSuggestions: {
          headerSubtitle: "\u4E00\u81F4\u3059\u308B\u30E1\u30FC\u30EB\u30C9\u30E1\u30A4\u30F3\u3092\u6301\u3064\u30E6\u30FC\u30B6\u30FC\u306F\u3001\u7D44\u7E54\u3078\u306E\u53C2\u52A0\u3092\u30EA\u30AF\u30A8\u30B9\u30C8\u3059\u308B\u63D0\u6848\u3092\u53D7\u3051\u53D6\u308B\u3053\u3068\u304C\u3067\u304D\u307E\u3059\u3002",
          headerTitle: "\u81EA\u52D5\u63D0\u6848",
          primaryButton: "\u691C\u8A3C\u6E08\u307F\u30C9\u30E1\u30A4\u30F3\u3092\u7BA1\u7406"
        },
        menuAction__approve: "\u627F\u8A8D",
        menuAction__reject: "\u62D2\u5426",
        tableHeader__requested: "\u30A2\u30AF\u30BB\u30B9\u3092\u30EA\u30AF\u30A8\u30B9\u30C8\u3057\u307E\u3057\u305F",
        table__emptyRow: "\u8868\u793A\u3059\u308B\u30EA\u30AF\u30A8\u30B9\u30C8\u306F\u3042\u308A\u307E\u305B\u3093"
      },
      start: {
        headerTitle__invitations: "\u62DB\u5F85",
        headerTitle__members: "\u30E1\u30F3\u30D0\u30FC",
        headerTitle__requests: "\u30EA\u30AF\u30A8\u30B9\u30C8"
      }
    },
    navbar: {
      description: "\u7D44\u7E54\u3092\u7BA1\u7406\u3057\u307E\u3059\u3002",
      general: "\u4E00\u822C",
      members: "\u30E1\u30F3\u30D0\u30FC",
      title: "\u7D44\u7E54"
    },
    profilePage: {
      dangerSection: {
        deleteOrganization: {
          actionDescription: '\u7D9A\u884C\u3059\u308B\u306B\u306F "{{organizationName}}" \u3068\u5165\u529B\u3057\u3066\u304F\u3060\u3055\u3044\u3002',
          messageLine1: "\u3053\u306E\u7D44\u7E54\u3092\u524A\u9664\u3057\u3066\u3082\u3088\u308D\u3057\u3044\u3067\u3059\u304B\uFF1F",
          messageLine2: "\u3053\u306E\u64CD\u4F5C\u306F\u6C38\u4E45\u7684\u3067\u53D6\u308A\u6D88\u3059\u3053\u3068\u306F\u3067\u304D\u307E\u305B\u3093\u3002",
          successMessage: "\u7D44\u7E54\u304C\u524A\u9664\u3055\u308C\u307E\u3057\u305F\u3002",
          title: "\u7D44\u7E54\u306E\u524A\u9664"
        },
        leaveOrganization: {
          actionDescription: '\u7D9A\u884C\u3059\u308B\u306B\u306F "{{organizationName}}" \u3068\u5165\u529B\u3057\u3066\u304F\u3060\u3055\u3044\u3002',
          messageLine1: "\u3053\u306E\u7D44\u7E54\u304B\u3089\u8131\u9000\u3057\u3066\u3082\u3088\u308D\u3057\u3044\u3067\u3059\u304B\uFF1F\u3053\u306E\u7D44\u7E54\u3068\u305D\u306E\u30A2\u30D7\u30EA\u30B1\u30FC\u30B7\u30E7\u30F3\u3078\u306E\u30A2\u30AF\u30BB\u30B9\u304C\u5931\u308F\u308C\u307E\u3059\u3002",
          messageLine2: "\u3053\u306E\u64CD\u4F5C\u306F\u6C38\u4E45\u7684\u3067\u53D6\u308A\u6D88\u3059\u3053\u3068\u306F\u3067\u304D\u307E\u305B\u3093\u3002",
          successMessage: "\u7D44\u7E54\u304B\u3089\u8131\u9000\u3057\u307E\u3057\u305F\u3002",
          title: "\u7D44\u7E54\u3092\u8131\u9000"
        },
        title: "\u6CE8\u610F"
      },
      domainSection: {
        menuAction__manage: "\u7BA1\u7406",
        menuAction__remove: "\u524A\u9664",
        menuAction__verify: "\u691C\u8A3C",
        primaryButton: "\u30C9\u30E1\u30A4\u30F3\u3092\u8FFD\u52A0",
        subtitle: "\u691C\u8A3C\u6E08\u307F\u306E\u30E1\u30FC\u30EB\u30C9\u30E1\u30A4\u30F3\u306B\u57FA\u3065\u3044\u3066\u3001\u30E6\u30FC\u30B6\u30FC\u304C\u81EA\u52D5\u7684\u306B\u7D44\u7E54\u306B\u53C2\u52A0\u3059\u308B\u304B\u3001\u53C2\u52A0\u3092\u30EA\u30AF\u30A8\u30B9\u30C8\u3059\u308B\u3053\u3068\u3092\u8A31\u53EF\u3057\u307E\u3059\u3002",
        title: "\u691C\u8A3C\u6E08\u307F\u30C9\u30E1\u30A4\u30F3"
      },
      successMessage: "\u7D44\u7E54\u304C\u66F4\u65B0\u3055\u308C\u307E\u3057\u305F\u3002",
      title: "\u7D44\u7E54\u30D7\u30ED\u30D5\u30A3\u30FC\u30EB"
    },
    removeDomainPage: {
      messageLine1: "\u30E1\u30FC\u30EB\u30C9\u30E1\u30A4\u30F3 {{domain}} \u304C\u524A\u9664\u3055\u308C\u307E\u3059\u3002",
      messageLine2: "\u3053\u306E\u5F8C\u3001\u30E6\u30FC\u30B6\u30FC\u306F\u81EA\u52D5\u7684\u306B\u7D44\u7E54\u306B\u53C2\u52A0\u3059\u308B\u3053\u3068\u304C\u3067\u304D\u306A\u304F\u306A\u308A\u307E\u3059\u3002",
      successMessage: "{{domain}} \u304C\u524A\u9664\u3055\u308C\u307E\u3057\u305F\u3002",
      title: "\u30C9\u30E1\u30A4\u30F3\u306E\u524A\u9664"
    },
    start: {
      headerTitle__general: "\u4E00\u822C",
      headerTitle__members: "\u30E1\u30F3\u30D0\u30FC",
      profileSection: {
        primaryButton: void 0,
        title: "\u7D44\u7E54\u30D7\u30ED\u30D5\u30A3\u30FC\u30EB",
        uploadAction__title: "\u30ED\u30B4"
      }
    },
    verifiedDomainPage: {
      dangerTab: {
        calloutInfoLabel: "\u3053\u306E\u30C9\u30E1\u30A4\u30F3\u3092\u524A\u9664\u3059\u308B\u3068\u3001\u62DB\u5F85\u3055\u308C\u305F\u30E6\u30FC\u30B6\u30FC\u306B\u5F71\u97FF\u304C\u51FA\u307E\u3059\u3002",
        removeDomainActionLabel__remove: "\u30C9\u30E1\u30A4\u30F3\u3092\u524A\u9664",
        removeDomainSubtitle: "\u691C\u8A3C\u6E08\u307F\u30C9\u30E1\u30A4\u30F3\u304B\u3089\u3053\u306E\u30C9\u30E1\u30A4\u30F3\u3092\u524A\u9664\u3057\u307E\u3059",
        removeDomainTitle: "\u30C9\u30E1\u30A4\u30F3\u306E\u524A\u9664"
      },
      enrollmentTab: {
        automaticInvitationOption__description: "\u30B5\u30A4\u30F3\u30A2\u30C3\u30D7\u6642\u306B\u30E6\u30FC\u30B6\u30FC\u306F\u81EA\u52D5\u7684\u306B\u7D44\u7E54\u306B\u62DB\u5F85\u3055\u308C\u3001\u3044\u3064\u3067\u3082\u53C2\u52A0\u3059\u308B\u3053\u3068\u304C\u3067\u304D\u307E\u3059\u3002",
        automaticInvitationOption__label: "\u81EA\u52D5\u62DB\u5F85",
        automaticSuggestionOption__description: "\u30E6\u30FC\u30B6\u30FC\u306F\u7D44\u7E54\u3078\u306E\u53C2\u52A0\u3092\u30EA\u30AF\u30A8\u30B9\u30C8\u3059\u308B\u63D0\u6848\u3092\u53D7\u3051\u53D6\u308A\u307E\u3059\u304C\u3001\u53C2\u52A0\u3059\u308B\u524D\u306B\u7BA1\u7406\u8005\u306E\u627F\u8A8D\u304C\u5FC5\u8981\u3067\u3059\u3002",
        automaticSuggestionOption__label: "\u81EA\u52D5\u63D0\u6848",
        calloutInfoLabel: "\u65B0\u3057\u3044\u30E6\u30FC\u30B6\u30FC\u306B\u306E\u307F\u767B\u9332\u30E2\u30FC\u30C9\u306E\u5909\u66F4\u304C\u5F71\u97FF\u3057\u307E\u3059\u3002",
        calloutInvitationCountLabel: "\u30E6\u30FC\u30B6\u30FC\u306B\u9001\u4FE1\u3055\u308C\u305F\u4FDD\u7559\u4E2D\u306E\u62DB\u5F85\u72B6: {{count}}",
        calloutSuggestionCountLabel: "\u30E6\u30FC\u30B6\u30FC\u306B\u9001\u4FE1\u3055\u308C\u305F\u4FDD\u7559\u4E2D\u306E\u63D0\u6848: {{count}}",
        manualInvitationOption__description: "\u30E6\u30FC\u30B6\u30FC\u306F\u7D44\u7E54\u306B\u624B\u52D5\u3067\u62DB\u5F85\u3055\u308C\u308B\u3053\u3068\u306E\u307F\u304C\u53EF\u80FD\u3067\u3059\u3002",
        manualInvitationOption__label: "\u81EA\u52D5\u767B\u9332\u306A\u3057",
        subtitle: "\u3053\u306E\u30C9\u30E1\u30A4\u30F3\u306E\u30E6\u30FC\u30B6\u30FC\u304C\u7D44\u7E54\u306B\u53C2\u52A0\u3059\u308B\u65B9\u6CD5\u3092\u9078\u629E\u3057\u3066\u304F\u3060\u3055\u3044\u3002"
      },
      start: {
        headerTitle__danger: "\u5371\u967A",
        headerTitle__enrollment: "\u767B\u9332\u30AA\u30D7\u30B7\u30E7\u30F3"
      },
      subtitle: "\u30C9\u30E1\u30A4\u30F3 {{domain}} \u304C\u691C\u8A3C\u3055\u308C\u307E\u3057\u305F\u3002\u767B\u9332\u30E2\u30FC\u30C9\u3092\u9078\u629E\u3057\u3066\u7D9A\u884C\u3057\u3066\u304F\u3060\u3055\u3044\u3002",
      title: "{{domain}} \u306E\u66F4\u65B0"
    },
    verifyDomainPage: {
      formSubtitle: "\u30E1\u30FC\u30EB\u30A2\u30C9\u30EC\u30B9\u306B\u9001\u4FE1\u3055\u308C\u305F\u691C\u8A3C\u30B3\u30FC\u30C9\u3092\u5165\u529B\u3057\u3066\u304F\u3060\u3055\u3044",
      formTitle: "\u691C\u8A3C\u30B3\u30FC\u30C9",
      resendButton: "\u30B3\u30FC\u30C9\u3092\u518D\u9001\u4FE1",
      subtitle: "\u30C9\u30E1\u30A4\u30F3 {{domainName}} \u306F\u30E1\u30FC\u30EB\u3067\u691C\u8A3C\u3059\u308B\u5FC5\u8981\u304C\u3042\u308A\u307E\u3059\u3002",
      subtitleVerificationCodeScreen: "{{emailAddress}} \u306B\u691C\u8A3C\u30B3\u30FC\u30C9\u304C\u9001\u4FE1\u3055\u308C\u307E\u3057\u305F\u3002\u30B3\u30FC\u30C9\u3092\u5165\u529B\u3057\u3066\u7D9A\u884C\u3057\u3066\u304F\u3060\u3055\u3044\u3002",
      title: "\u30C9\u30E1\u30A4\u30F3\u3092\u691C\u8A3C"
    }
  },
  organizationSwitcher: {
    action__createOrganization: "\u7D44\u7E54\u306E\u4F5C\u6210",
    action__invitationAccept: "\u53C2\u52A0\u3059\u308B",
    action__manageOrganization: "\u7D44\u7E54\u306E\u7BA1\u7406",
    action__suggestionsAccept: "\u53C2\u52A0\u3092\u30EA\u30AF\u30A8\u30B9\u30C8\u3059\u308B",
    notSelected: "\u7D44\u7E54\u304C\u9078\u629E\u3055\u308C\u3066\u3044\u307E\u305B\u3093",
    personalWorkspace: "\u500B\u4EBA\u30EF\u30FC\u30AF\u30B9\u30DA\u30FC\u30B9",
    suggestionsAcceptedLabel: "\u627F\u8A8D\u5F85\u3061"
  },
  paginationButton__next: "\u6B21\u3078",
  paginationButton__previous: "\u524D\u3078",
  paginationRowText__displaying: "\u8868\u793A\u4E2D",
  paginationRowText__of: "\u5168",
  signIn: {
    accountSwitcher: {
      action__addAccount: "\u30A2\u30AB\u30A6\u30F3\u30C8\u3092\u8FFD\u52A0",
      action__signOutAll: "\u5168\u3066\u306E\u30A2\u30AB\u30A6\u30F3\u30C8\u304B\u3089\u30B5\u30A4\u30F3\u30A2\u30A6\u30C8",
      subtitle: "\u7D9A\u884C\u3059\u308B\u30A2\u30AB\u30A6\u30F3\u30C8\u3092\u9078\u629E\u3057\u3066\u304F\u3060\u3055\u3044\u3002",
      title: "\u30A2\u30AB\u30A6\u30F3\u30C8\u3092\u9078\u629E"
    },
    alternativeMethods: {
      actionLink: "\u30D8\u30EB\u30D7\u3092\u53D6\u5F97",
      actionText: "\u3053\u308C\u3089\u306E\u3044\u305A\u308C\u3082\u6301\u3063\u3066\u3044\u307E\u305B\u3093\u304B\uFF1F",
      blockButton__backupCode: "\u30D0\u30C3\u30AF\u30A2\u30C3\u30D7\u30B3\u30FC\u30C9\u3092\u4F7F\u7528\u3059\u308B",
      blockButton__emailCode: "{{identifier}}\u306B\u30E1\u30FC\u30EB\u30B3\u30FC\u30C9\u3092\u9001\u4FE1",
      blockButton__emailLink: "{{identifier}}\u306B\u30E1\u30FC\u30EB\u30EA\u30F3\u30AF\u3092\u9001\u4FE1",
      blockButton__passkey: void 0,
      blockButton__password: "\u30D1\u30B9\u30EF\u30FC\u30C9\u3067\u30B5\u30A4\u30F3\u30A4\u30F3\u3059\u308B",
      blockButton__phoneCode: "{{identifier}}\u306BSMS\u30B3\u30FC\u30C9\u3092\u9001\u4FE1",
      blockButton__totp: "\u8A8D\u8A3C\u30A2\u30D7\u30EA\u3092\u4F7F\u7528\u3059\u308B",
      getHelp: {
        blockButton__emailSupport: "\u30E1\u30FC\u30EB\u30B5\u30DD\u30FC\u30C8",
        content: "\u30A2\u30AB\u30A6\u30F3\u30C8\u306B\u30B5\u30A4\u30F3\u30A4\u30F3\u3067\u304D\u306A\u3044\u5834\u5408\u306F\u3001\u30E1\u30FC\u30EB\u3067\u304A\u554F\u3044\u5408\u308F\u305B\u3044\u305F\u3060\u3051\u308C\u3070\u3001\u3067\u304D\u308B\u3060\u3051\u65E9\u304F\u30A2\u30AF\u30BB\u30B9\u3092\u56DE\u5FA9\u3059\u308B\u305F\u3081\u306B\u304A\u624B\u4F1D\u3044\u3044\u305F\u3057\u307E\u3059\u3002",
        title: "\u30D8\u30EB\u30D7\u3092\u53D6\u5F97"
      },
      subtitle: "\u554F\u984C\u304C\u767A\u751F\u3057\u3066\u3044\u307E\u3059\u304B\uFF1F\u3053\u308C\u3089\u306E\u65B9\u6CD5\u3092\u4F7F\u7528\u3057\u3066\u30B5\u30A4\u30F3\u30A4\u30F3\u3059\u308B\u3053\u3068\u304C\u3067\u304D\u307E\u3059\u3002",
      title: "\u5225\u306E\u65B9\u6CD5\u3092\u4F7F\u7528"
    },
    backupCodeMfa: {
      subtitle: "{{applicationName}}\u3078\u306E\u30A2\u30AF\u30BB\u30B9\u3092\u7D9A\u3051\u308B",
      title: "\u30D0\u30C3\u30AF\u30A2\u30C3\u30D7\u30B3\u30FC\u30C9\u3092\u5165\u529B"
    },
    emailCode: {
      formTitle: "\u691C\u8A3C\u30B3\u30FC\u30C9",
      resendButton: "\u30B3\u30FC\u30C9\u3092\u518D\u9001\u4FE1",
      subtitle: "{{applicationName}}\u3078\u306E\u30A2\u30AF\u30BB\u30B9\u3092\u7D9A\u3051\u308B",
      title: "\u30E1\u30FC\u30EB\u3092\u78BA\u8A8D"
    },
    emailLink: {
      clientMismatch: {
        subtitle: void 0,
        title: void 0
      },
      expired: {
        subtitle: "\u5143\u306E\u30BF\u30D6\u306B\u623B\u3063\u3066\u7D9A\u884C\u3057\u3066\u304F\u3060\u3055\u3044\u3002",
        title: "\u3053\u306E\u691C\u8A3C\u30EA\u30F3\u30AF\u306F\u671F\u9650\u5207\u308C\u3067\u3059"
      },
      failed: {
        subtitle: "\u5143\u306E\u30BF\u30D6\u306B\u623B\u3063\u3066\u7D9A\u884C\u3057\u3066\u304F\u3060\u3055\u3044\u3002",
        title: "\u3053\u306E\u691C\u8A3C\u30EA\u30F3\u30AF\u306F\u7121\u52B9\u3067\u3059"
      },
      formSubtitle: "\u30E1\u30FC\u30EB\u306B\u9001\u4FE1\u3055\u308C\u305F\u691C\u8A3C\u30EA\u30F3\u30AF\u3092\u4F7F\u7528\u3057\u3066\u304F\u3060\u3055\u3044",
      formTitle: "\u691C\u8A3C\u30EA\u30F3\u30AF",
      loading: {
        subtitle: "\u307E\u3082\u306A\u304F\u30EA\u30C0\u30A4\u30EC\u30AF\u30C8\u3055\u308C\u307E\u3059",
        title: "\u30B5\u30A4\u30F3\u30A4\u30F3\u4E2D..."
      },
      resendButton: "\u30EA\u30F3\u30AF\u3092\u518D\u9001\u4FE1",
      subtitle: "{{applicationName}}\u3078\u306E\u30A2\u30AF\u30BB\u30B9\u3092\u7D9A\u3051\u308B",
      title: "\u30E1\u30FC\u30EB\u3092\u78BA\u8A8D",
      unusedTab: {
        title: "\u3053\u306E\u30BF\u30D6\u3092\u9589\u3058\u3066\u3082\u304B\u307E\u3044\u307E\u305B\u3093"
      },
      verified: {
        subtitle: "\u307E\u3082\u306A\u304F\u30EA\u30C0\u30A4\u30EC\u30AF\u30C8\u3055\u308C\u307E\u3059",
        title: "\u6B63\u5E38\u306B\u30B5\u30A4\u30F3\u30A4\u30F3\u3057\u307E\u3057\u305F"
      },
      verifiedSwitchTab: {
        subtitle: "\u7D9A\u884C\u3059\u308B\u306B\u306F\u5143\u306E\u30BF\u30D6\u306B\u623B\u3063\u3066\u304F\u3060\u3055\u3044",
        subtitleNewTab: "\u65B0\u3057\u304F\u958B\u3044\u305F\u30BF\u30D6\u306B\u623B\u3063\u3066\u7D9A\u884C\u3057\u3066\u304F\u3060\u3055\u3044",
        titleNewTab: "\u4ED6\u306E\u30BF\u30D6\u3067\u30B5\u30A4\u30F3\u30A4\u30F3\u6E08\u307F"
      }
    },
    forgotPassword: {
      formTitle: "\u30D1\u30B9\u30EF\u30FC\u30C9\u30EA\u30BB\u30C3\u30C8\u30B3\u30FC\u30C9",
      resendButton: "\u30B3\u30FC\u30C9\u3092\u518D\u9001\u4FE1",
      subtitle: "\u30D1\u30B9\u30EF\u30FC\u30C9\u3092\u30EA\u30BB\u30C3\u30C8\u3059\u308B\u305F\u3081\u306B",
      subtitle_email: "\u307E\u305A\u3001\u30E1\u30FC\u30EBID\u306B\u9001\u4FE1\u3055\u308C\u305F\u30B3\u30FC\u30C9\u3092\u5165\u529B\u3057\u3066\u304F\u3060\u3055\u3044",
      subtitle_phone: "\u307E\u305A\u3001\u96FB\u8A71\u306B\u9001\u4FE1\u3055\u308C\u305F\u30B3\u30FC\u30C9\u3092\u5165\u529B\u3057\u3066\u304F\u3060\u3055\u3044",
      title: "\u30D1\u30B9\u30EF\u30FC\u30C9\u3092\u30EA\u30BB\u30C3\u30C8"
    },
    forgotPasswordAlternativeMethods: {
      blockButton__resetPassword: "\u30D1\u30B9\u30EF\u30FC\u30C9\u3092\u30EA\u30BB\u30C3\u30C8",
      label__alternativeMethods: "\u307E\u305F\u306F\u3001\u5225\u306E\u65B9\u6CD5\u3067\u30B5\u30A4\u30F3\u30A4\u30F3\u3057\u3066\u304F\u3060\u3055\u3044\u3002",
      title: "\u30D1\u30B9\u30EF\u30FC\u30C9\u3092\u304A\u5FD8\u308C\u3067\u3059\u304B\uFF1F"
    },
    noAvailableMethods: {
      message: "\u30B5\u30A4\u30F3\u30A4\u30F3\u3067\u304D\u307E\u305B\u3093\u3002\u5229\u7528\u53EF\u80FD\u306A\u8A8D\u8A3C\u65B9\u6CD5\u304C\u3042\u308A\u307E\u305B\u3093\u3002",
      subtitle: "\u30A8\u30E9\u30FC\u304C\u767A\u751F\u3057\u307E\u3057\u305F",
      title: "\u30B5\u30A4\u30F3\u30A4\u30F3\u3067\u304D\u307E\u305B\u3093"
    },
    passkey: {
      subtitle: void 0,
      title: void 0
    },
    password: {
      actionLink: "\u5225\u306E\u65B9\u6CD5\u3092\u4F7F\u7528",
      subtitle: "{{applicationName}}\u3078\u306E\u30A2\u30AF\u30BB\u30B9\u3092\u7D9A\u3051\u308B",
      title: "\u30D1\u30B9\u30EF\u30FC\u30C9\u3092\u5165\u529B"
    },
    passwordPwned: {
      title: void 0
    },
    phoneCode: {
      formTitle: "\u691C\u8A3C\u30B3\u30FC\u30C9",
      resendButton: "\u30B3\u30FC\u30C9\u3092\u518D\u9001\u4FE1",
      subtitle: "{{applicationName}} \u3078\u306E\u7D9A\u884C\u306E\u305F\u3081",
      title: "\u96FB\u8A71\u3092\u78BA\u8A8D\u3057\u3066\u304F\u3060\u3055\u3044"
    },
    phoneCodeMfa: {
      formTitle: "\u691C\u8A3C\u30B3\u30FC\u30C9",
      resendButton: "\u30B3\u30FC\u30C9\u3092\u518D\u9001\u4FE1",
      subtitle: void 0,
      title: "\u96FB\u8A71\u3092\u78BA\u8A8D\u3057\u3066\u304F\u3060\u3055\u3044"
    },
    resetPassword: {
      formButtonPrimary: "\u30D1\u30B9\u30EF\u30FC\u30C9\u3092\u30EA\u30BB\u30C3\u30C8",
      requiredMessage: "\u672A\u78BA\u8A8D\u306E\u30E1\u30FC\u30EB\u30A2\u30C9\u30EC\u30B9\u3092\u6301\u3064\u30A2\u30AB\u30A6\u30F3\u30C8\u304C\u65E2\u306B\u5B58\u5728\u3057\u307E\u3059\u3002\u30BB\u30AD\u30E5\u30EA\u30C6\u30A3\u306E\u305F\u3081\u306B\u30D1\u30B9\u30EF\u30FC\u30C9\u3092\u30EA\u30BB\u30C3\u30C8\u3057\u3066\u304F\u3060\u3055\u3044\u3002",
      successMessage: "\u30D1\u30B9\u30EF\u30FC\u30C9\u304C\u6B63\u5E38\u306B\u5909\u66F4\u3055\u308C\u307E\u3057\u305F\u3002\u304A\u5F85\u3061\u304F\u3060\u3055\u3044\u3001\u30B5\u30A4\u30F3\u30A4\u30F3\u3057\u3066\u3044\u307E\u3059\u3002",
      title: "\u30D1\u30B9\u30EF\u30FC\u30C9\u3092\u30EA\u30BB\u30C3\u30C8"
    },
    resetPasswordMfa: {
      detailsLabel: "\u30D1\u30B9\u30EF\u30FC\u30C9\u3092\u30EA\u30BB\u30C3\u30C8\u3059\u308B\u524D\u306B\u3001\u8EAB\u5143\u3092\u78BA\u8A8D\u3059\u308B\u5FC5\u8981\u304C\u3042\u308A\u307E\u3059\u3002"
    },
    start: {
      actionLink: "\u30B5\u30A4\u30F3\u30A2\u30C3\u30D7",
      actionLink__use_email: "\u30E1\u30FC\u30EB\u30A2\u30C9\u30EC\u30B9\u3092\u4F7F\u7528",
      actionLink__use_email_username: "\u30E1\u30FC\u30EB\u30A2\u30C9\u30EC\u30B9\u307E\u305F\u306F\u30E6\u30FC\u30B6\u30FC\u540D\u3092\u4F7F\u7528",
      actionLink__use_passkey: void 0,
      actionLink__use_phone: "\u96FB\u8A71\u756A\u53F7\u3092\u4F7F\u7528",
      actionLink__use_username: "\u30E6\u30FC\u30B6\u30FC\u540D\u3092\u4F7F\u7528",
      actionText: "\u30A2\u30AB\u30A6\u30F3\u30C8\u3092\u304A\u6301\u3061\u3067\u306A\u3044\u3067\u3059\u304B\uFF1F",
      subtitle: "{{applicationName}}\u3078\u306E\u30A2\u30AF\u30BB\u30B9\u3092\u7D9A\u3051\u308B",
      title: "\u30B5\u30A4\u30F3\u30A4\u30F3"
    },
    totpMfa: {
      formTitle: "\u691C\u8A3C\u30B3\u30FC\u30C9",
      subtitle: void 0,
      title: "\u4E8C\u6BB5\u968E\u8A8D\u8A3C"
    }
  },
  signInEnterPasswordTitle: "\u30D1\u30B9\u30EF\u30FC\u30C9\u3092\u5165\u529B\u3057\u3066\u304F\u3060\u3055\u3044",
  signUp: {
    continue: {
      actionLink: "\u30B5\u30A4\u30F3\u30A4\u30F3",
      actionText: "\u30A2\u30AB\u30A6\u30F3\u30C8\u3092\u304A\u6301\u3061\u3067\u3059\u304B\uFF1F",
      subtitle: "{{applicationName}}\u3078\u306E\u30A2\u30AF\u30BB\u30B9\u3092\u7D9A\u3051\u308B",
      title: "\u672A\u5165\u529B\u306E\u30D5\u30A3\u30FC\u30EB\u30C9\u3092\u5165\u529B"
    },
    emailCode: {
      formSubtitle: "\u30E1\u30FC\u30EB\u30A2\u30C9\u30EC\u30B9\u306B\u9001\u4FE1\u3055\u308C\u305F\u78BA\u8A8D\u30B3\u30FC\u30C9\u3092\u5165\u529B\u3057\u3066\u304F\u3060\u3055\u3044",
      formTitle: "\u78BA\u8A8D\u30B3\u30FC\u30C9",
      resendButton: "\u30B3\u30FC\u30C9\u3092\u518D\u9001\u4FE1",
      subtitle: "{{applicationName}}\u3078\u306E\u30A2\u30AF\u30BB\u30B9\u3092\u7D9A\u3051\u308B",
      title: "\u30E1\u30FC\u30EB\u30A2\u30C9\u30EC\u30B9\u3092\u78BA\u8A8D"
    },
    emailLink: {
      clientMismatch: {
        subtitle: void 0,
        title: void 0
      },
      formSubtitle: "\u30E1\u30FC\u30EB\u30A2\u30C9\u30EC\u30B9\u306B\u9001\u4FE1\u3055\u308C\u305F\u78BA\u8A8D\u30EA\u30F3\u30AF\u3092\u4F7F\u7528\u3057\u3066\u304F\u3060\u3055\u3044",
      formTitle: "\u78BA\u8A8D\u30EA\u30F3\u30AF",
      loading: {
        title: "\u767B\u9332\u4E2D..."
      },
      resendButton: "\u30EA\u30F3\u30AF\u3092\u518D\u9001\u4FE1",
      subtitle: "{{applicationName}}\u3078\u306E\u30A2\u30AF\u30BB\u30B9\u3092\u7D9A\u3051\u308B",
      title: "\u30E1\u30FC\u30EB\u30A2\u30C9\u30EC\u30B9\u3092\u78BA\u8A8D",
      verified: {
        title: "\u767B\u9332\u304C\u5B8C\u4E86\u3057\u307E\u3057\u305F"
      },
      verifiedSwitchTab: {
        subtitle: "\u7D9A\u884C\u3059\u308B\u305F\u3081\u306B\u65B0\u3057\u304F\u958B\u3044\u305F\u30BF\u30D6\u306B\u623B\u3063\u3066\u304F\u3060\u3055\u3044",
        subtitleNewTab: "\u7D9A\u884C\u3059\u308B\u305F\u3081\u306B\u524D\u306E\u30BF\u30D6\u306B\u623B\u3063\u3066\u304F\u3060\u3055\u3044",
        title: "\u30E1\u30FC\u30EB\u30A2\u30C9\u30EC\u30B9\u304C\u6B63\u5E38\u306B\u78BA\u8A8D\u3055\u308C\u307E\u3057\u305F"
      }
    },
    phoneCode: {
      formSubtitle: "\u96FB\u8A71\u756A\u53F7\u306B\u9001\u4FE1\u3055\u308C\u305F\u78BA\u8A8D\u30B3\u30FC\u30C9\u3092\u5165\u529B\u3057\u3066\u304F\u3060\u3055\u3044",
      formTitle: "\u78BA\u8A8D\u30B3\u30FC\u30C9",
      resendButton: "\u30B3\u30FC\u30C9\u3092\u518D\u9001\u4FE1",
      subtitle: "{{applicationName}}\u3078\u306E\u30A2\u30AF\u30BB\u30B9\u3092\u7D9A\u3051\u308B",
      title: "\u96FB\u8A71\u756A\u53F7\u3092\u78BA\u8A8D"
    },
    start: {
      actionLink: "\u30B5\u30A4\u30F3\u30A4\u30F3",
      actionLink__use_email: void 0,
      actionLink__use_phone: void 0,
      actionText: "\u30A2\u30AB\u30A6\u30F3\u30C8\u3092\u304A\u6301\u3061\u3067\u3059\u304B\uFF1F",
      subtitle: "{{applicationName}}\u3078\u306E\u30A2\u30AF\u30BB\u30B9\u3092\u7D9A\u3051\u308B",
      title: "\u30A2\u30AB\u30A6\u30F3\u30C8\u3092\u4F5C\u6210"
    }
  },
  socialButtonsBlockButton: "{{provider|titleize}}\u3067\u7D9A\u3051\u308B",
  socialButtonsBlockButtonManyInView: void 0,
  unstable__errors: {
    already_a_member_in_organization: void 0,
    captcha_invalid: "\u30BB\u30AD\u30E5\u30EA\u30C6\u30A3\u691C\u8A3C\u306B\u5931\u6557\u3057\u305F\u305F\u3081\u3001\u30B5\u30A4\u30F3\u30A2\u30C3\u30D7\u306B\u5931\u6557\u3057\u307E\u3057\u305F\u3002\u30DA\u30FC\u30B8\u3092\u66F4\u65B0\u3057\u3066\u518D\u8A66\u884C\u3059\u308B\u304B\u3001\u30B5\u30DD\u30FC\u30C8\u30BB\u30F3\u30BF\u30FC\u306B\u9023\u7D61\u3057\u305F\u4E0A\u3067\u30B5\u30DD\u30FC\u30C8\u3092\u53D7\u3051\u3066\u304F\u3060\u3055\u3044\u3002",
    captcha_unavailable: "\u30DC\u30C3\u30C8\u691C\u8A3C\u306B\u5931\u6557\u3057\u305F\u305F\u3081\u3001\u30B5\u30A4\u30F3\u30A2\u30C3\u30D7\u306B\u5931\u6557\u3057\u307E\u3057\u305F\u3002\u30DA\u30FC\u30B8\u3092\u66F4\u65B0\u3057\u3066\u518D\u8A66\u884C\u3059\u308B\u304B\u3001\u30B5\u30DD\u30FC\u30C8\u306B\u9023\u7D61\u3057\u3066\u3055\u3089\u306B\u652F\u63F4\u3092\u53D7\u3051\u3066\u304F\u3060\u3055\u3044\u3002",
    form_code_incorrect: "\u30D5\u30A9\u30FC\u30E0\u30B3\u30FC\u30C9\u304C\u6B63\u3057\u304F\u3042\u308A\u307E\u305B\u3093\u3002",
    form_identifier_exists: "\u8B58\u5225\u5B50\u304C\u65E2\u306B\u5B58\u5728\u3057\u307E\u3059\u3002",
    form_identifier_exists__email_address: void 0,
    form_identifier_exists__phone_number: void 0,
    form_identifier_exists__username: void 0,
    form_identifier_not_found: "\u8B58\u5225\u5B50\u304C\u898B\u3064\u304B\u308A\u307E\u305B\u3093\u3002",
    form_param_format_invalid: "\u30D1\u30E9\u30E1\u30FC\u30BF\u5F62\u5F0F\u304C\u7121\u52B9\u3067\u3059\u3002",
    form_param_format_invalid__email_address: "\u30E1\u30FC\u30EB\u30A2\u30C9\u30EC\u30B9\u306F\u6709\u52B9\u306A\u30E1\u30FC\u30EB\u30A2\u30C9\u30EC\u30B9\u3067\u3042\u308B\u5FC5\u8981\u304C\u3042\u308A\u307E\u3059\u3002",
    form_param_format_invalid__phone_number: "\u96FB\u8A71\u756A\u53F7\u306F\u6709\u52B9\u306A\u56FD\u969B\u5F62\u5F0F\u3067\u3042\u308B\u5FC5\u8981\u304C\u3042\u308A\u307E\u3059",
    form_param_max_length_exceeded__first_name: "\u6C0F\u540D\u306F256\u6587\u5B57\u3092\u8D85\u3048\u308B\u3053\u3068\u306F\u3067\u304D\u307E\u305B\u3093\u3002",
    form_param_max_length_exceeded__last_name: "\u59D3\u306F256\u6587\u5B57\u3092\u8D85\u3048\u308B\u3053\u3068\u306F\u3067\u304D\u307E\u305B\u3093\u3002",
    form_param_max_length_exceeded__name: "\u540D\u524D\u306F256\u6587\u5B57\u3092\u8D85\u3048\u308B\u3053\u3068\u306F\u3067\u304D\u307E\u305B\u3093\u3002",
    form_param_nil: "\u30D1\u30E9\u30E1\u30FC\u30BF\u304C\u5B58\u5728\u3057\u307E\u305B\u3093\u3002",
    form_param_value_invalid: void 0,
    form_password_incorrect: "\u30D1\u30B9\u30EF\u30FC\u30C9\u304C\u6B63\u3057\u304F\u3042\u308A\u307E\u305B\u3093\u3002",
    form_password_length_too_short: "\u30D1\u30B9\u30EF\u30FC\u30C9\u306E\u9577\u3055\u304C\u77ED\u3059\u304E\u307E\u3059\u3002",
    form_password_not_strong_enough: "\u30D1\u30B9\u30EF\u30FC\u30C9\u306E\u5F37\u5EA6\u304C\u4E0D\u5341\u5206\u3067\u3059\u3002",
    form_password_pwned: "\u3053\u306E\u30D1\u30B9\u30EF\u30FC\u30C9\u306F\u4FB5\u5BB3\u306E\u4E00\u90E8\u3068\u3057\u3066\u898B\u3064\u304B\u3063\u305F\u305F\u3081\u4F7F\u7528\u3067\u304D\u307E\u305B\u3093\u3002\u5225\u306E\u30D1\u30B9\u30EF\u30FC\u30C9\u3092\u8A66\u3057\u3066\u304F\u3060\u3055\u3044\u3002",
    form_password_pwned__sign_in: void 0,
    form_password_size_in_bytes_exceeded: "\u30D1\u30B9\u30EF\u30FC\u30C9\u306E\u30D0\u30A4\u30C8\u6570\u304C\u4E0A\u9650\u3092\u8D85\u3048\u3066\u3044\u307E\u3059\u3002\u77ED\u304F\u3059\u308B\u304B\u3001\u4E00\u90E8\u306E\u7279\u6B8A\u6587\u5B57\u3092\u524A\u9664\u3057\u3066\u304F\u3060\u3055\u3044\u3002",
    form_password_validation_failed: "\u30D1\u30B9\u30EF\u30FC\u30C9\u304C\u9593\u9055\u3063\u3066\u3044\u307E\u3059",
    form_username_invalid_character: "\u30E6\u30FC\u30B6\u30FC\u540D\u306B\u7121\u52B9\u306A\u6587\u5B57\u304C\u542B\u307E\u308C\u3066\u3044\u307E\u3059\u3002",
    form_username_invalid_length: "\u30E6\u30FC\u30B6\u30FC\u540D\u306E\u9577\u3055\u304C\u7121\u52B9\u3067\u3059\u3002",
    identification_deletion_failed: "\u6700\u5F8C\u306E\u8B58\u5225\u60C5\u5831\u306F\u524A\u9664\u3067\u304D\u307E\u305B\u3093\u3002",
    not_allowed_access: "\u30A2\u30AF\u30BB\u30B9\u304C\u8A31\u53EF\u3055\u308C\u3066\u3044\u307E\u305B\u3093\u3002",
    organization_domain_blocked: void 0,
    organization_domain_common: void 0,
    organization_membership_quota_exceeded: void 0,
    organization_minimum_permissions_needed: void 0,
    passkey_already_exists: void 0,
    passkey_not_supported: void 0,
    passkey_pa_not_supported: void 0,
    passkey_registration_cancelled: void 0,
    passkey_retrieval_cancelled: void 0,
    passwordComplexity: {
      maximumLength: "{{length}}\u6587\u5B57\u672A\u6E80",
      minimumLength: "{{length}}\u6587\u5B57\u4EE5\u4E0A",
      requireLowercase: "\u5C0F\u6587\u5B57\u3092\u542B\u3080",
      requireNumbers: "\u6570\u5B57\u3092\u542B\u3080",
      requireSpecialCharacter: "\u7279\u6B8A\u6587\u5B57\u3092\u542B\u3080",
      requireUppercase: "\u5927\u6587\u5B57\u3092\u542B\u3080",
      sentencePrefix: "\u30D1\u30B9\u30EF\u30FC\u30C9\u306F\u6B21\u306E\u6761\u4EF6\u3092\u6E80\u305F\u3059\u5FC5\u8981\u304C\u3042\u308A\u307E\u3059\uFF1A"
    },
    phone_number_exists: "\u3053\u306E\u96FB\u8A71\u756A\u53F7\u306F\u65E2\u306B\u4F7F\u7528\u3055\u308C\u3066\u3044\u307E\u3059\u3002\u5225\u306E\u96FB\u8A71\u756A\u53F7\u3092\u8A66\u3057\u3066\u304F\u3060\u3055\u3044\u3002",
    zxcvbn: {
      couldBeStronger: "\u30D1\u30B9\u30EF\u30FC\u30C9\u306F\u6709\u52B9\u3067\u3059\u304C\u3001\u3082\u3046\u5C11\u3057\u5F37\u5316\u3067\u304D\u307E\u3059\u3002\u6587\u5B57\u3092\u8FFD\u52A0\u3057\u3066\u307F\u3066\u304F\u3060\u3055\u3044\u3002",
      goodPassword: "\u7D20\u6674\u3089\u3057\u3044\u4ED5\u4E8B\u3067\u3059\u3002\u3053\u308C\u306F\u512A\u308C\u305F\u30D1\u30B9\u30EF\u30FC\u30C9\u3067\u3059\u3002",
      notEnough: "\u30D1\u30B9\u30EF\u30FC\u30C9\u306E\u5F37\u5EA6\u304C\u5341\u5206\u3067\u306F\u3042\u308A\u307E\u305B\u3093\u3002",
      suggestions: {
        allUppercase: "\u5168\u3066\u306E\u6587\u5B57\u3092\u5927\u6587\u5B57\u306B\u3059\u308B\u306E\u3067\u306F\u306A\u304F\u3001\u4E00\u90E8\u306E\u6587\u5B57\u3092\u5927\u6587\u5B57\u306B\u3057\u3066\u304F\u3060\u3055\u3044\u3002",
        anotherWord: "\u3088\u308A\u4E00\u822C\u7684\u3067\u306A\u3044\u5358\u8A9E\u3092\u8FFD\u52A0\u3057\u3066\u304F\u3060\u3055\u3044\u3002",
        associatedYears: "\u81EA\u5206\u306B\u95A2\u9023\u3059\u308B\u5E74\u53F7\u306F\u907F\u3051\u3066\u304F\u3060\u3055\u3044\u3002",
        capitalization: "\u6700\u521D\u306E\u6587\u5B57\u4EE5\u5916\u3082\u5927\u6587\u5B57\u306B\u3057\u3066\u304F\u3060\u3055\u3044\u3002",
        dates: "\u81EA\u5206\u306B\u95A2\u9023\u3059\u308B\u65E5\u4ED8\u3084\u5E74\u53F7\u306F\u907F\u3051\u3066\u304F\u3060\u3055\u3044\u3002",
        l33t: "\u4E88\u6E2C\u53EF\u80FD\u306A\u6587\u5B57\u306E\u4EE3\u66FF\uFF08\u4F8B\uFF1A'@' \u3067 'a' \u3092\u7F6E\u304D\u63DB\u3048\u308B\uFF09\u3092\u907F\u3051\u3066\u304F\u3060\u3055\u3044\u3002",
        longerKeyboardPattern: "\u9577\u3044\u30AD\u30FC\u30DC\u30FC\u30C9\u30D1\u30BF\u30FC\u30F3\u3092\u4F7F\u7528\u3057\u3001\u30BF\u30A4\u30D4\u30F3\u30B0\u306E\u65B9\u5411\u3092\u8907\u6570\u56DE\u5909\u3048\u3066\u304F\u3060\u3055\u3044\u3002",
        noNeed: "\u30B7\u30F3\u30DC\u30EB\u3001\u6570\u5B57\u3001\u5927\u6587\u5B57\u306E\u4F7F\u7528\u306A\u3057\u3067\u3082\u5F37\u529B\u306A\u30D1\u30B9\u30EF\u30FC\u30C9\u3092\u4F5C\u6210\u3067\u304D\u307E\u3059\u3002",
        pwned: "\u4ED6\u306E\u5834\u6240\u3067\u3053\u306E\u30D1\u30B9\u30EF\u30FC\u30C9\u3092\u4F7F\u7528\u3057\u3066\u3044\u308B\u5834\u5408\u306F\u3001\u5909\u66F4\u3059\u308B\u5FC5\u8981\u304C\u3042\u308A\u307E\u3059\u3002",
        recentYears: "\u6700\u8FD1\u306E\u5E74\u53F7\u306F\u907F\u3051\u3066\u304F\u3060\u3055\u3044\u3002",
        repeated: "\u7E70\u308A\u8FD4\u3055\u308C\u308B\u5358\u8A9E\u3084\u6587\u5B57\u3092\u907F\u3051\u3066\u304F\u3060\u3055\u3044\u3002",
        reverseWords: "\u4E00\u822C\u7684\u306A\u5358\u8A9E\u306E\u9006\u3055\u8AAD\u307F\u306F\u907F\u3051\u3066\u304F\u3060\u3055\u3044\u3002",
        sequences: "\u4E00\u822C\u7684\u306A\u6587\u5B57\u306E\u4E26\u3073\u3092\u907F\u3051\u3066\u304F\u3060\u3055\u3044\u3002",
        useWords: "\u8907\u6570\u306E\u5358\u8A9E\u3092\u4F7F\u7528\u3057\u3066\u304F\u3060\u3055\u3044\u304C\u3001\u4E00\u822C\u7684\u306A\u30D5\u30EC\u30FC\u30BA\u306F\u907F\u3051\u3066\u304F\u3060\u3055\u3044\u3002"
      },
      warnings: {
        common: "\u3053\u308C\u306F\u4E00\u822C\u7684\u306B\u4F7F\u308F\u308C\u308B\u30D1\u30B9\u30EF\u30FC\u30C9\u3067\u3059\u3002",
        commonNames: "\u4E00\u822C\u7684\u306A\u540D\u524D\u3084\u59D3\u306F\u63A8\u6E2C\u3057\u3084\u3059\u3044\u3067\u3059\u3002",
        dates: "\u65E5\u4ED8\u306F\u63A8\u6E2C\u3057\u3084\u3059\u3044\u3067\u3059\u3002",
        extendedRepeat: "\u300Cabcabcabc\u300D\u3068\u3044\u3063\u305F\u7E70\u308A\u8FD4\u3057\u30D1\u30BF\u30FC\u30F3\u306F\u63A8\u6E2C\u3057\u3084\u3059\u3044\u3067\u3059\u3002",
        keyPattern: "\u77ED\u3044\u30AD\u30FC\u30DC\u30FC\u30C9\u30D1\u30BF\u30FC\u30F3\u306F\u63A8\u6E2C\u3057\u3084\u3059\u3044\u3067\u3059\u3002",
        namesByThemselves: "\u5358\u4F53\u306E\u540D\u524D\u3084\u59D3\u306F\u63A8\u6E2C\u3057\u3084\u3059\u3044\u3067\u3059\u3002",
        pwned: "\u3053\u306E\u30D1\u30B9\u30EF\u30FC\u30C9\u306F\u30A4\u30F3\u30BF\u30FC\u30CD\u30C3\u30C8\u4E0A\u306E\u30C7\u30FC\u30BF\u4FB5\u5BB3\u306B\u3088\u3063\u3066\u516C\u958B\u3055\u308C\u3066\u3044\u307E\u3059\u3002",
        recentYears: "\u6700\u8FD1\u306E\u5E74\u53F7\u306F\u63A8\u6E2C\u3057\u3084\u3059\u3044\u3067\u3059\u3002",
        sequences: "\u300Cabc\u300D\u3068\u3044\u3063\u305F\u4E00\u822C\u7684\u306A\u6587\u5B57\u306E\u4E26\u3073\u306F\u63A8\u6E2C\u3057\u3084\u3059\u3044\u3067\u3059\u3002",
        similarToCommon: "\u3053\u308C\u306F\u4E00\u822C\u7684\u306B\u4F7F\u308F\u308C\u308B\u30D1\u30B9\u30EF\u30FC\u30C9\u306B\u985E\u4F3C\u3057\u3066\u3044\u307E\u3059\u3002",
        simpleRepeat: "\u300Caaa\u300D\u3068\u3044\u3063\u305F\u7E70\u308A\u8FD4\u3057\u6587\u5B57\u306F\u63A8\u6E2C\u3057\u3084\u3059\u3044\u3067\u3059\u3002",
        straightRow: "\u30AD\u30FC\u30DC\u30FC\u30C9\u4E0A\u306E\u9023\u7D9A\u3057\u305F\u884C\u306F\u63A8\u6E2C\u3057\u3084\u3059\u3044\u3067\u3059\u3002",
        topHundred: "\u3053\u308C\u306F\u983B\u7E41\u306B\u4F7F\u308F\u308C\u308B\u30D1\u30B9\u30EF\u30FC\u30C9\u3067\u3059\u3002",
        topTen: "\u3053\u308C\u306F\u3088\u304F\u4F7F\u308F\u308C\u308B\u30D1\u30B9\u30EF\u30FC\u30C9\u3067\u3059\u3002",
        userInputs: "\u500B\u4EBA\u60C5\u5831\u3084\u30DA\u30FC\u30B8\u306B\u95A2\u9023\u3059\u308B\u30C7\u30FC\u30BF\u306F\u542B\u307E\u308C\u3066\u3044\u306A\u3044\u306F\u305A\u3067\u3059\u3002",
        wordByItself: "\u5358\u8A9E\u5358\u4F53\u3067\u306F\u63A8\u6E2C\u3057\u3084\u3059\u3044\u3067\u3059\u3002"
      }
    }
  },
  userButton: {
    action__addAccount: "\u30A2\u30AB\u30A6\u30F3\u30C8\u306E\u8FFD\u52A0",
    action__manageAccount: "\u30A2\u30AB\u30A6\u30F3\u30C8\u306E\u7BA1\u7406",
    action__signOut: "\u30B5\u30A4\u30F3\u30A2\u30A6\u30C8",
    action__signOutAll: "\u5168\u3066\u306E\u30A2\u30AB\u30A6\u30F3\u30C8\u304B\u3089\u30B5\u30A4\u30F3\u30A2\u30A6\u30C8"
  },
  userProfile: {
    backupCodePage: {
      actionLabel__copied: "\u30B3\u30D4\u30FC\u6E08\u307F\uFF01",
      actionLabel__copy: "\u3059\u3079\u3066\u30B3\u30D4\u30FC",
      actionLabel__download: ".txt\u3067\u30C0\u30A6\u30F3\u30ED\u30FC\u30C9",
      actionLabel__print: "\u5370\u5237",
      infoText1: "\u3053\u306E\u30A2\u30AB\u30A6\u30F3\u30C8\u3067\u306F\u30D0\u30C3\u30AF\u30A2\u30C3\u30D7\u30B3\u30FC\u30C9\u304C\u6709\u52B9\u306B\u306A\u308A\u307E\u3059\u3002",
      infoText2: "\u30D0\u30C3\u30AF\u30A2\u30C3\u30D7\u30B3\u30FC\u30C9\u306F\u79D8\u5BC6\u306B\u4FDD\u7BA1\u3057\u3001\u5B89\u5168\u306B\u4FDD\u5B58\u3057\u3066\u304F\u3060\u3055\u3044\u3002\u7591\u308F\u3057\u3044\u5834\u5408\u306F\u30D0\u30C3\u30AF\u30A2\u30C3\u30D7\u30B3\u30FC\u30C9\u3092\u518D\u751F\u6210\u3059\u308B\u3053\u3068\u304C\u3067\u304D\u307E\u3059\u3002",
      subtitle__codelist: "\u30D0\u30C3\u30AF\u30A2\u30C3\u30D7\u30B3\u30FC\u30C9\u3092\u5B89\u5168\u306B\u4FDD\u7BA1\u3057\u3001\u79D8\u5BC6\u306B\u3057\u3066\u304F\u3060\u3055\u3044\u3002",
      successMessage: "\u30D0\u30C3\u30AF\u30A2\u30C3\u30D7\u30B3\u30FC\u30C9\u304C\u6709\u52B9\u306B\u306A\u308A\u307E\u3057\u305F\u3002\u8A8D\u8A3C\u30C7\u30D0\u30A4\u30B9\u306B\u30A2\u30AF\u30BB\u30B9\u3067\u304D\u306A\u3044\u5834\u5408\u3001\u3053\u308C\u3089\u306E\u3044\u305A\u308C\u304B\u3092\u4F7F\u7528\u3057\u3066\u30A2\u30AB\u30A6\u30F3\u30C8\u306B\u30B5\u30A4\u30F3\u30A4\u30F3\u3067\u304D\u307E\u3059\u3002\u5404\u30B3\u30FC\u30C9\u306F\u4E00\u5EA6\u3057\u304B\u4F7F\u7528\u3067\u304D\u307E\u305B\u3093\u3002",
      successSubtitle: "\u8A8D\u8A3C\u30C7\u30D0\u30A4\u30B9\u306B\u30A2\u30AF\u30BB\u30B9\u3067\u304D\u306A\u3044\u5834\u5408\u3001\u3053\u308C\u3089\u306E\u3044\u305A\u308C\u304B\u3092\u4F7F\u7528\u3057\u3066\u30A2\u30AB\u30A6\u30F3\u30C8\u306B\u30B5\u30A4\u30F3\u30A4\u30F3\u3067\u304D\u307E\u3059\u3002",
      title: "\u30D0\u30C3\u30AF\u30A2\u30C3\u30D7\u30B3\u30FC\u30C9\u306E\u8FFD\u52A0",
      title__codelist: "\u30D0\u30C3\u30AF\u30A2\u30C3\u30D7\u30B3\u30FC\u30C9"
    },
    connectedAccountPage: {
      formHint: "\u30A2\u30AB\u30A6\u30F3\u30C8\u3092\u9023\u643A\u3059\u308B\u30D7\u30ED\u30D0\u30A4\u30C0\u3092\u9078\u629E\u3057\u3066\u304F\u3060\u3055\u3044\u3002",
      formHint__noAccounts: "\u5229\u7528\u53EF\u80FD\u306A\u5916\u90E8\u30A2\u30AB\u30A6\u30F3\u30C8\u30D7\u30ED\u30D0\u30A4\u30C0\u306F\u3042\u308A\u307E\u305B\u3093\u3002",
      removeResource: {
        messageLine1: "{{identifier}}\u306F\u3053\u306E\u30A2\u30AB\u30A6\u30F3\u30C8\u304B\u3089\u524A\u9664\u3055\u308C\u307E\u3059\u3002",
        messageLine2: "\u3053\u306E\u9023\u643A\u30A2\u30AB\u30A6\u30F3\u30C8\u3092\u4F7F\u7528\u3059\u308B\u3053\u3068\u306F\u3067\u304D\u306A\u304F\u306A\u308A\u3001\u95A2\u9023\u3059\u308B\u6A5F\u80FD\u3082\u4F7F\u7528\u3067\u304D\u306A\u304F\u306A\u308A\u307E\u3059\u3002",
        successMessage: "{{connectedAccount}}\u304C\u30A2\u30AB\u30A6\u30F3\u30C8\u304B\u3089\u524A\u9664\u3055\u308C\u307E\u3057\u305F\u3002",
        title: "\u9023\u643A\u30A2\u30AB\u30A6\u30F3\u30C8\u306E\u524A\u9664"
      },
      socialButtonsBlockButton: "{{provider|titleize}}\u30A2\u30AB\u30A6\u30F3\u30C8\u3092\u9023\u643A\u3059\u308B",
      successMessage: "\u30D7\u30ED\u30D0\u30A4\u30C0\u304C\u30A2\u30AB\u30A6\u30F3\u30C8\u306B\u8FFD\u52A0\u3055\u308C\u307E\u3057\u305F",
      title: "\u9023\u643A\u30A2\u30AB\u30A6\u30F3\u30C8\u306E\u8FFD\u52A0"
    },
    deletePage: {
      actionDescription: "\u7D9A\u884C\u3059\u308B\u306B\u306F\u4E0B\u8A18\u306B\u300CDelete account\u300D\u3092\u5165\u529B\u3057\u3066\u304F\u3060\u3055\u3044\u3002",
      confirm: "\u30A2\u30AB\u30A6\u30F3\u30C8\u524A\u9664",
      messageLine1: "\u30A2\u30AB\u30A6\u30F3\u30C8\u3092\u524A\u9664\u3057\u3066\u3082\u3088\u308D\u3057\u3044\u3067\u3059\u304B\uFF1F",
      messageLine2: "\u3053\u306E\u64CD\u4F5C\u306F\u6C38\u4E45\u7684\u3067\u53D6\u308A\u6D88\u3059\u3053\u3068\u306F\u3067\u304D\u307E\u305B\u3093\u3002",
      title: "\u30A2\u30AB\u30A6\u30F3\u30C8\u306E\u524A\u9664"
    },
    emailAddressPage: {
      emailCode: {
        formHint: "\u3053\u306E\u30E1\u30FC\u30EB\u30A2\u30C9\u30EC\u30B9\u306B\u306F\u691C\u8A3C\u30B3\u30FC\u30C9\u304C\u542B\u307E\u308C\u305F\u30E1\u30FC\u30EB\u304C\u9001\u4FE1\u3055\u308C\u307E\u3059\u3002",
        formSubtitle: "{{identifier}}\u306B\u9001\u4FE1\u3055\u308C\u305F\u691C\u8A3C\u30B3\u30FC\u30C9\u3092\u5165\u529B\u3057\u3066\u304F\u3060\u3055\u3044\u3002",
        formTitle: "\u691C\u8A3C\u30B3\u30FC\u30C9",
        resendButton: "\u30B3\u30FC\u30C9\u3092\u518D\u9001\u4FE1",
        successMessage: "\u30E1\u30FC\u30EB\u30A2\u30C9\u30EC\u30B9{{identifier}}\u304C\u30A2\u30AB\u30A6\u30F3\u30C8\u306B\u8FFD\u52A0\u3055\u308C\u307E\u3057\u305F\u3002"
      },
      emailLink: {
        formHint: "\u3053\u306E\u30E1\u30FC\u30EB\u30A2\u30C9\u30EC\u30B9\u306B\u306F\u691C\u8A3C\u30EA\u30F3\u30AF\u304C\u542B\u307E\u308C\u305F\u30E1\u30FC\u30EB\u304C\u9001\u4FE1\u3055\u308C\u307E\u3059\u3002",
        formSubtitle: "{{identifier}}\u306B\u9001\u4FE1\u3055\u308C\u305F\u30E1\u30FC\u30EB\u306E\u691C\u8A3C\u30EA\u30F3\u30AF\u3092\u30AF\u30EA\u30C3\u30AF\u3057\u3066\u304F\u3060\u3055\u3044\u3002",
        formTitle: "\u691C\u8A3C\u30EA\u30F3\u30AF",
        resendButton: "\u30EA\u30F3\u30AF\u3092\u518D\u9001\u4FE1",
        successMessage: "\u30E1\u30FC\u30EB\u30A2\u30C9\u30EC\u30B9{{identifier}}\u304C\u30A2\u30AB\u30A6\u30F3\u30C8\u306B\u8FFD\u52A0\u3055\u308C\u307E\u3057\u305F\u3002"
      },
      removeResource: {
        messageLine1: "{{identifier}}\u306F\u3053\u306E\u30A2\u30AB\u30A6\u30F3\u30C8\u304B\u3089\u524A\u9664\u3055\u308C\u307E\u3059\u3002",
        messageLine2: "\u3053\u306E\u30E1\u30FC\u30EB\u30A2\u30C9\u30EC\u30B9\u3092\u4F7F\u7528\u3057\u3066\u306E\u30B5\u30A4\u30F3\u30A4\u30F3\u306F\u3067\u304D\u306A\u304F\u306A\u308A\u307E\u3059\u3002",
        successMessage: "{{emailAddress}}\u304C\u30A2\u30AB\u30A6\u30F3\u30C8\u304B\u3089\u524A\u9664\u3055\u308C\u307E\u3057\u305F\u3002",
        title: "\u30E1\u30FC\u30EB\u30A2\u30C9\u30EC\u30B9\u306E\u524A\u9664"
      },
      title: "\u30E1\u30FC\u30EB\u30A2\u30C9\u30EC\u30B9\u306E\u8FFD\u52A0",
      verifyTitle: "\u30E1\u30FC\u30EB\u30A2\u30C9\u30EC\u30B9\u306E\u78BA\u8A8D"
    },
    formButtonPrimary__add: "\u8FFD\u52A0",
    formButtonPrimary__continue: "\u7D9A\u884C",
    formButtonPrimary__finish: "\u5B8C\u4E86",
    formButtonPrimary__remove: "\u524A\u9664",
    formButtonPrimary__save: "\u4FDD\u5B58",
    formButtonReset: "\u30AD\u30E3\u30F3\u30BB\u30EB",
    mfaPage: {
      formHint: "\u8FFD\u52A0\u3059\u308B\u65B9\u6CD5\u3092\u9078\u629E\u3057\u3066\u304F\u3060\u3055\u3044\u3002",
      title: "\u4E8C\u6BB5\u968E\u8A8D\u8A3C\u306E\u8FFD\u52A0"
    },
    mfaPhoneCodePage: {
      backButton: "\u65E2\u5B58\u306E\u756A\u53F7\u3092\u4F7F\u7528",
      primaryButton__addPhoneNumber: "\u96FB\u8A71\u756A\u53F7\u3092\u8FFD\u52A0",
      removeResource: {
        messageLine1: "{{identifier}}\u306F\u30B5\u30A4\u30F3\u30A4\u30F3\u6642\u306B\u8A8D\u8A3C\u30B3\u30FC\u30C9\u3092\u53D7\u3051\u53D6\u3089\u306A\u304F\u306A\u308A\u307E\u3059\u3002",
        messageLine2: "\u30A2\u30AB\u30A6\u30F3\u30C8\u306E\u30BB\u30AD\u30E5\u30EA\u30C6\u30A3\u304C\u4F4E\u4E0B\u3059\u308B\u53EF\u80FD\u6027\u304C\u3042\u308A\u307E\u3059\u3002\u672C\u5F53\u306B\u524A\u9664\u3057\u307E\u3059\u304B\uFF1F",
        successMessage: "{{mfaPhoneCode}}\u306ESMS\u30B3\u30FC\u30C9\u4E8C\u6BB5\u968E\u8A8D\u8A3C\u304C\u524A\u9664\u3055\u308C\u307E\u3057\u305F\u3002",
        title: "\u4E8C\u6BB5\u968E\u8A8D\u8A3C\u306E\u524A\u9664"
      },
      subtitle__availablePhoneNumbers: "SMS\u30B3\u30FC\u30C9\u4E8C\u6BB5\u968E\u8A8D\u8A3C\u306E\u305F\u3081\u306B\u767B\u9332\u3059\u308B\u96FB\u8A71\u756A\u53F7\u3092\u9078\u629E\u3057\u3066\u304F\u3060\u3055\u3044\u3002",
      subtitle__unavailablePhoneNumbers: "SMS\u30B3\u30FC\u30C9\u4E8C\u6BB5\u968E\u8A8D\u8A3C\u306E\u305F\u3081\u306B\u5229\u7528\u53EF\u80FD\u306A\u96FB\u8A71\u756A\u53F7\u306F\u3042\u308A\u307E\u305B\u3093\u3002",
      successMessage1: "When signing in, you will need to enter a verification code sent to this phone number as an additional step.",
      successMessage2: "Save these backup codes and store them somewhere safe. If you lose access to your authentication device, you can use backup codes to sign in.",
      successTitle: "SMS\u30B3\u30FC\u30C9\u8A8D\u8A3C\u304C\u6709\u52B9\u306B\u306A\u308A\u307E\u3057\u305F",
      title: "SMS\u30B3\u30FC\u30C9\u8A8D\u8A3C\u306E\u8FFD\u52A0"
    },
    mfaTOTPPage: {
      authenticatorApp: {
        buttonAbleToScan__nonPrimary: "\u4EE3\u308F\u308A\u306BQR\u30B3\u30FC\u30C9\u3092\u30B9\u30AD\u30E3\u30F3\u3059\u308B",
        buttonUnableToScan__nonPrimary: "QR\u30B3\u30FC\u30C9\u3092\u30B9\u30AD\u30E3\u30F3\u3067\u304D\u307E\u305B\u3093\u304B\uFF1F",
        infoText__ableToScan: "\u8A8D\u8A3C\u30A2\u30D7\u30EA\u3067\u65B0\u3057\u3044\u30B5\u30A4\u30F3\u30A4\u30F3\u65B9\u6CD5\u3092\u8A2D\u5B9A\u3057\u3001\u4EE5\u4E0B\u306EQR\u30B3\u30FC\u30C9\u3092\u30B9\u30AD\u30E3\u30F3\u3057\u3066\u30A2\u30AB\u30A6\u30F3\u30C8\u3068\u30EA\u30F3\u30AF\u3055\u305B\u307E\u3059\u3002",
        infoText__unableToScan: "\u8A8D\u8A3C\u30A2\u30D7\u30EA\u3067\u65B0\u3057\u3044\u30B5\u30A4\u30F3\u30A4\u30F3\u65B9\u6CD5\u3092\u8A2D\u5B9A\u3057\u3001\u4EE5\u4E0B\u306E\u30AD\u30FC\u3092\u5165\u529B\u3057\u3066\u304F\u3060\u3055\u3044\u3002",
        inputLabel__unableToScan1: "\u30BF\u30A4\u30E0\u30D9\u30FC\u30B9\u307E\u305F\u306F\u30EF\u30F3\u30BF\u30A4\u30E0\u30D1\u30B9\u30EF\u30FC\u30C9\u304C\u6709\u52B9\u306B\u306A\u3063\u3066\u3044\u308B\u3053\u3068\u3092\u78BA\u8A8D\u3057\u3001\u30A2\u30AB\u30A6\u30F3\u30C8\u306E\u30EA\u30F3\u30AF\u3092\u5B8C\u4E86\u3057\u3066\u304F\u3060\u3055\u3044\u3002",
        inputLabel__unableToScan2: "\u307E\u305F\u3001\u8A8D\u8A3C\u30A2\u30D7\u30EA\u304CTOTP URI\u3092\u30B5\u30DD\u30FC\u30C8\u3057\u3066\u3044\u308B\u5834\u5408\u306F\u3001\u5B8C\u5168\u306AURI\u3092\u30B3\u30D4\u30FC\u3059\u308B\u3053\u3068\u3082\u3067\u304D\u307E\u3059\u3002"
      },
      removeResource: {
        messageLine1: "\u3053\u306E\u8A8D\u8A3C\u30A2\u30D7\u30EA\u304B\u3089\u306E\u691C\u8A3C\u30B3\u30FC\u30C9\u306F\u3001\u30B5\u30A4\u30F3\u30A4\u30F3\u6642\u306B\u306F\u4E0D\u8981\u306B\u306A\u308A\u307E\u3059\u3002",
        messageLine2: "\u30A2\u30AB\u30A6\u30F3\u30C8\u306E\u30BB\u30AD\u30E5\u30EA\u30C6\u30A3\u304C\u4F4E\u4E0B\u3059\u308B\u53EF\u80FD\u6027\u304C\u3042\u308A\u307E\u3059\u3002\u672C\u5F53\u306B\u524A\u9664\u3057\u307E\u3059\u304B\uFF1F",
        successMessage: "\u8A8D\u8A3C\u30A2\u30D7\u30EA\u3092\u4F7F\u7528\u3057\u305F\u4E8C\u6BB5\u968E\u8A8D\u8A3C\u304C\u524A\u9664\u3055\u308C\u307E\u3057\u305F\u3002",
        title: "\u4E8C\u6BB5\u968E\u8A8D\u8A3C\u306E\u524A\u9664"
      },
      successMessage: "\u4E8C\u6BB5\u968E\u8A8D\u8A3C\u304C\u6709\u52B9\u306B\u306A\u308A\u307E\u3057\u305F\u3002\u30B5\u30A4\u30F3\u30A4\u30F3\u6642\u306B\u306F\u3001\u3053\u306E\u8A8D\u8A3C\u30A2\u30D7\u30EA\u304B\u3089\u306E\u691C\u8A3C\u30B3\u30FC\u30C9\u3092\u8FFD\u52A0\u306E\u30B9\u30C6\u30C3\u30D7\u3068\u3057\u3066\u5165\u529B\u3059\u308B\u5FC5\u8981\u304C\u3042\u308A\u307E\u3059\u3002",
      title: "\u8A8D\u8A3C\u30A2\u30D7\u30EA\u306E\u8FFD\u52A0",
      verifySubtitle: "\u8A8D\u8A3C\u30A2\u30D7\u30EA\u3067\u751F\u6210\u3055\u308C\u305F\u691C\u8A3C\u30B3\u30FC\u30C9\u3092\u5165\u529B\u3057\u3066\u304F\u3060\u3055\u3044\u3002",
      verifyTitle: "\u691C\u8A3C\u30B3\u30FC\u30C9"
    },
    mobileButton__menu: "\u30E1\u30CB\u30E5\u30FC",
    navbar: {
      account: "\u30D7\u30ED\u30D5\u30A1\u30A4\u30EB",
      description: "\u30A2\u30AB\u30A6\u30F3\u30C8\u60C5\u5831\u7BA1\u7406",
      security: "\u30BB\u30AD\u30E5\u30EA\u30C6\u30A3",
      title: "\u30A2\u30AB\u30A6\u30F3\u30C8"
    },
    passkeyScreen: {
      removeResource: {
        messageLine1: void 0,
        title: void 0
      },
      subtitle__rename: void 0,
      title__rename: void 0
    },
    passwordPage: {
      checkboxInfoText__signOutOfOtherSessions: "\u53E4\u3044\u30D1\u30B9\u30EF\u30FC\u30C9\u3092\u4F7F\u7528\u3057\u3066\u3044\u308B\u53EF\u80FD\u6027\u306E\u3042\u308B\u3059\u3079\u3066\u306E\u30C7\u30D0\u30A4\u30B9\u304B\u3089\u30B5\u30A4\u30F3\u30A2\u30A6\u30C8\u3059\u308B\u3053\u3068\u3092\u304A\u52E7\u3081\u3057\u307E\u3059\u3002",
      readonly: "\u30D7\u30ED\u30D5\u30A1\u30A4\u30EB\u60C5\u5831\u306F\u30A8\u30F3\u30BF\u30FC\u30D7\u30E9\u30A4\u30BA\u63A5\u7D9A\u306B\u3088\u3063\u3066\u63D0\u4F9B\u3055\u308C\u3066\u304A\u308A\u3001\u7DE8\u96C6\u3067\u304D\u307E\u305B\u3093\u3002",
      successMessage__set: "\u30D1\u30B9\u30EF\u30FC\u30C9\u304C\u8A2D\u5B9A\u3055\u308C\u307E\u3057\u305F\u3002",
      successMessage__signOutOfOtherSessions: "\u4ED6\u306E\u3059\u3079\u3066\u306E\u30C7\u30D0\u30A4\u30B9\u304B\u3089\u30B5\u30A4\u30F3\u30A2\u30A6\u30C8\u3055\u308C\u307E\u3057\u305F\u3002",
      successMessage__update: "\u30D1\u30B9\u30EF\u30FC\u30C9\u304C\u66F4\u65B0\u3055\u308C\u307E\u3057\u305F\u3002",
      title__set: "\u30D1\u30B9\u30EF\u30FC\u30C9\u306E\u8A2D\u5B9A",
      title__update: "\u30D1\u30B9\u30EF\u30FC\u30C9\u306E\u66F4\u65B0"
    },
    phoneNumberPage: {
      infoText: "\u3053\u306E\u96FB\u8A71\u756A\u53F7\u306B\u306F\u691C\u8A3C\u30EA\u30F3\u30AF\u304C\u542B\u307E\u308C\u305F\u30C6\u30AD\u30B9\u30C8\u30E1\u30C3\u30BB\u30FC\u30B8\u304C\u9001\u4FE1\u3055\u308C\u307E\u3059\u3002",
      removeResource: {
        messageLine1: "{{identifier}}\u306F\u3053\u306E\u30A2\u30AB\u30A6\u30F3\u30C8\u304B\u3089\u524A\u9664\u3055\u308C\u307E\u3059\u3002",
        messageLine2: "\u3053\u306E\u96FB\u8A71\u756A\u53F7\u3092\u4F7F\u7528\u3057\u3066\u306E\u30B5\u30A4\u30F3\u30A4\u30F3\u306F\u3067\u304D\u306A\u304F\u306A\u308A\u307E\u3059\u3002",
        successMessage: "{{phoneNumber}}\u304C\u30A2\u30AB\u30A6\u30F3\u30C8\u304B\u3089\u524A\u9664\u3055\u308C\u307E\u3057\u305F\u3002",
        title: "\u96FB\u8A71\u756A\u53F7\u306E\u524A\u9664"
      },
      successMessage: "{{identifier}}\u304C\u30A2\u30AB\u30A6\u30F3\u30C8\u306B\u8FFD\u52A0\u3055\u308C\u307E\u3057\u305F\u3002",
      title: "\u96FB\u8A71\u756A\u53F7\u306E\u8FFD\u52A0",
      verifySubtitle: "{{identifier}}\u306B\u9001\u4FE1\u3055\u308C\u305F\u691C\u8A3C\u30B3\u30FC\u30C9\u3092\u5165\u529B\u3057\u3066\u304F\u3060\u3055\u3044",
      verifyTitle: "\u96FB\u8A71\u756A\u53F7\u306E\u78BA\u8A8D"
    },
    profilePage: {
      fileDropAreaHint: "10MB\u672A\u6E80\u306EJPG\u3001PNG\u3001GIF\u3001\u307E\u305F\u306FWEBP\u753B\u50CF\u3092\u30A2\u30C3\u30D7\u30ED\u30FC\u30C9\u3057\u3066\u304F\u3060\u3055\u3044",
      imageFormDestructiveActionSubtitle: "\u753B\u50CF\u306E\u524A\u9664",
      imageFormSubtitle: "\u753B\u50CF\u306E\u30A2\u30C3\u30D7\u30ED\u30FC\u30C9",
      imageFormTitle: "\u30D7\u30ED\u30D5\u30A3\u30FC\u30EB\u753B\u50CF",
      readonly: "\u30D7\u30ED\u30D5\u30A1\u30A4\u30EB\u60C5\u5831\u306F\u30A8\u30F3\u30BF\u30FC\u30D7\u30E9\u30A4\u30BA\u63A5\u7D9A\u306B\u3088\u3063\u3066\u63D0\u4F9B\u3055\u308C\u3066\u304A\u308A\u3001\u7DE8\u96C6\u3067\u304D\u307E\u305B\u3093\u3002",
      successMessage: "\u30D7\u30ED\u30D5\u30A3\u30FC\u30EB\u304C\u66F4\u65B0\u3055\u308C\u307E\u3057\u305F\u3002",
      title: "\u30D7\u30ED\u30D5\u30A3\u30FC\u30EB\u306E\u66F4\u65B0"
    },
    start: {
      activeDevicesSection: {
        destructiveAction: "\u30C7\u30D0\u30A4\u30B9\u304B\u3089\u30B5\u30A4\u30F3\u30A2\u30A6\u30C8",
        title: "\u30A2\u30AF\u30C6\u30A3\u30D6\u306A\u30C7\u30D0\u30A4\u30B9"
      },
      connectedAccountsSection: {
        actionLabel__connectionFailed: "\u518D\u8A66\u884C",
        actionLabel__reauthorize: "\u4ECA\u3059\u3050\u8A8D\u8A3C",
        destructiveActionTitle: "\u524A\u9664",
        primaryButton: "\u30A2\u30AB\u30A6\u30F3\u30C8\u3092\u9023\u643A\u3059\u308B",
        subtitle__disconnected: void 0,
        subtitle__reauthorize: "\u5FC5\u8981\u306A\u30B9\u30B3\u30FC\u30D7\u304C\u66F4\u65B0\u3055\u308C\u3001\u6A5F\u80FD\u304C\u5236\u9650\u3055\u308C\u3066\u3044\u308B\u53EF\u80FD\u6027\u304C\u3042\u308A\u307E\u3059\u3002\u554F\u984C\u3092\u907F\u3051\u308B\u305F\u3081\u306B\u3001\u3053\u306E\u30A2\u30D7\u30EA\u30B1\u30FC\u30B7\u30E7\u30F3\u3092\u518D\u8A8D\u8A3C\u3057\u3066\u304F\u3060\u3055\u3044\u3002",
        title: "\u9023\u643A\u30A2\u30AB\u30A6\u30F3\u30C8"
      },
      dangerSection: {
        deleteAccountButton: "\u30A2\u30AB\u30A6\u30F3\u30C8\u306E\u524A\u9664",
        title: "\u30A2\u30AB\u30A6\u30F3\u30C8\u306E\u7D42\u4E86"
      },
      emailAddressesSection: {
        destructiveAction: "\u30E1\u30FC\u30EB\u30A2\u30C9\u30EC\u30B9\u306E\u524A\u9664",
        detailsAction__nonPrimary: "\u30D7\u30E9\u30A4\u30DE\u30EA\u306B\u8A2D\u5B9A\u3059\u308B",
        detailsAction__primary: "\u78BA\u8A8D\u3092\u5B8C\u4E86\u3059\u308B",
        detailsAction__unverified: "\u78BA\u8A8D\u3092\u5B8C\u4E86\u3059\u308B",
        primaryButton: "\u30E1\u30FC\u30EB\u30A2\u30C9\u30EC\u30B9\u306E\u8FFD\u52A0",
        title: "\u30E1\u30FC\u30EB\u30A2\u30C9\u30EC\u30B9"
      },
      enterpriseAccountsSection: {
        title: "\u30A8\u30F3\u30BF\u30FC\u30D7\u30E9\u30A4\u30BA\u30A2\u30AB\u30A6\u30F3\u30C8"
      },
      headerTitle__account: "\u30A2\u30AB\u30A6\u30F3\u30C8",
      headerTitle__security: "\u30BB\u30AD\u30E5\u30EA\u30C6\u30A3",
      mfaSection: {
        backupCodes: {
          actionLabel__regenerate: "\u30B3\u30FC\u30C9\u3092\u518D\u751F\u6210",
          headerTitle: "\u30D0\u30C3\u30AF\u30A2\u30C3\u30D7\u30B3\u30FC\u30C9",
          subtitle__regenerate: "\u5B89\u5168\u306A\u65B0\u3057\u3044\u30D0\u30C3\u30AF\u30A2\u30C3\u30D7\u30B3\u30FC\u30C9\u3092\u53D6\u5F97\u3057\u307E\u3059\u3002\u4EE5\u524D\u306E\u30D0\u30C3\u30AF\u30A2\u30C3\u30D7\u30B3\u30FC\u30C9\u306F\u524A\u9664\u3055\u308C\u3001\u4F7F\u7528\u3059\u308B\u3053\u3068\u306F\u3067\u304D\u307E\u305B\u3093\u3002",
          title__regenerate: "\u30D0\u30C3\u30AF\u30A2\u30C3\u30D7\u30B3\u30FC\u30C9\u306E\u518D\u751F\u6210"
        },
        phoneCode: {
          actionLabel__setDefault: "\u30C7\u30D5\u30A9\u30EB\u30C8\u306B\u8A2D\u5B9A",
          destructiveActionLabel: "\u96FB\u8A71\u756A\u53F7\u306E\u524A\u9664"
        },
        primaryButton: "\u4E8C\u6BB5\u968E\u8A8D\u8A3C\u3092\u8FFD\u52A0\u3059\u308B",
        title: "\u4E8C\u6BB5\u968E\u8A8D\u8A3C",
        totp: {
          destructiveActionTitle: "\u524A\u9664",
          headerTitle: "\u8A8D\u8A3C\u30A2\u30D7\u30EA\u30B1\u30FC\u30B7\u30E7\u30F3"
        }
      },
      passkeysSection: {
        menuAction__destructive: void 0,
        menuAction__rename: void 0,
        title: void 0
      },
      passwordSection: {
        primaryButton__setPassword: "\u30D1\u30B9\u30EF\u30FC\u30C9\u3092\u8A2D\u5B9A\u3059\u308B",
        primaryButton__updatePassword: "\u30D1\u30B9\u30EF\u30FC\u30C9\u3092\u5909\u66F4\u3059\u308B",
        title: "\u30D1\u30B9\u30EF\u30FC\u30C9"
      },
      phoneNumbersSection: {
        destructiveAction: "\u96FB\u8A71\u756A\u53F7\u306E\u524A\u9664",
        detailsAction__nonPrimary: "\u30D7\u30E9\u30A4\u30DE\u30EA\u306B\u8A2D\u5B9A\u3059\u308B",
        detailsAction__primary: "\u78BA\u8A8D\u3092\u5B8C\u4E86\u3059\u308B",
        detailsAction__unverified: "\u78BA\u8A8D\u3092\u5B8C\u4E86\u3059\u308B",
        primaryButton: "\u96FB\u8A71\u756A\u53F7\u306E\u8FFD\u52A0",
        title: "\u96FB\u8A71\u756A\u53F7"
      },
      profileSection: {
        primaryButton: void 0,
        title: "\u30D7\u30ED\u30D5\u30A3\u30FC\u30EB"
      },
      usernameSection: {
        primaryButton__setUsername: "\u30E6\u30FC\u30B6\u30FC\u540D\u306E\u8A2D\u5B9A",
        primaryButton__updateUsername: "\u30E6\u30FC\u30B6\u30FC\u540D\u306E\u5909\u66F4",
        title: "\u30E6\u30FC\u30B6\u30FC\u540D"
      },
      web3WalletsSection: {
        destructiveAction: "\u30A6\u30A9\u30EC\u30C3\u30C8\u306E\u524A\u9664",
        primaryButton: "Web3\u30A6\u30A9\u30EC\u30C3\u30C8",
        title: "Web3\u30A6\u30A9\u30EC\u30C3\u30C8"
      }
    },
    usernamePage: {
      successMessage: "\u30E6\u30FC\u30B6\u30FC\u540D\u304C\u66F4\u65B0\u3055\u308C\u307E\u3057\u305F\u3002",
      title__set: "\u30E6\u30FC\u30B6\u30FC\u540D\u306E\u66F4\u65B0",
      title__update: "\u30E6\u30FC\u30B6\u30FC\u540D\u306E\u66F4\u65B0"
    },
    web3WalletPage: {
      removeResource: {
        messageLine1: "{{identifier}}\u306F\u3053\u306E\u30A2\u30AB\u30A6\u30F3\u30C8\u304B\u3089\u524A\u9664\u3055\u308C\u307E\u3059\u3002",
        messageLine2: "\u3053\u306EWeb3\u30A6\u30A9\u30EC\u30C3\u30C8\u3092\u4F7F\u7528\u3057\u3066\u306E\u30B5\u30A4\u30F3\u30A4\u30F3\u306F\u3067\u304D\u306A\u304F\u306A\u308A\u307E\u3059\u3002",
        successMessage: "{{web3Wallet}}\u304C\u30A2\u30AB\u30A6\u30F3\u30C8\u304B\u3089\u524A\u9664\u3055\u308C\u307E\u3057\u305F\u3002",
        title: "Web3\u30A6\u30A9\u30EC\u30C3\u30C8\u306E\u524A\u9664"
      },
      subtitle__availableWallets: "\u30A2\u30AB\u30A6\u30F3\u30C8\u306B\u63A5\u7D9A\u3059\u308BWeb3\u30A6\u30A9\u30EC\u30C3\u30C8\u3092\u9078\u629E\u3057\u3066\u304F\u3060\u3055\u3044\u3002",
      subtitle__unavailableWallets: "\u5229\u7528\u53EF\u80FD\u306AWeb3\u30A6\u30A9\u30EC\u30C3\u30C8\u306F\u3042\u308A\u307E\u305B\u3093\u3002",
      successMessage: "\u30A6\u30A9\u30EC\u30C3\u30C8\u304C\u30A2\u30AB\u30A6\u30F3\u30C8\u306B\u8FFD\u52A0\u3055\u308C\u307E\u3057\u305F\u3002",
      title: "Web3\u30A6\u30A9\u30EC\u30C3\u30C8\u306E\u8FFD\u52A0",
      web3WalletButtonsBlockButton: void 0
    }
  }
};
// Annotate the CommonJS export names for ESM import in node:
0 && (module.exports = {
  jaJP
});
